code/modules/admin/verbs/adminpm.dm

```diff
@@ -50,7 +52,9 @@
 		else
 			targets["(No Mob) - [T]"] = T
 	var/list/sorted = sortList(targets)
-	var/target = input(src,"To whom shall we send a message?","Admin PM",null) in sorted|null
+	var/target = input(src,"To whom shall we send a message?","Admin PM",null) as null|anything in sorted
```

Comment:
Is the `sorted|null` not needed anymore?

Reply:
its still there, just this is more performant